lib/plugins/inspec-suggest/test/functional/suggtool_bump_test.rb

```diff
@@ -0,0 +1,32 @@
+require_relative "../helper"
+require "tmpdir" unless defined?(Dir.mktmpdir)
+
+describe "inspec suggtool bump" do
+  include CorePluginFunctionalHelper
+
+  let(:fixture_path) { File.join(repo_path, "lib", "plugins", "inspec-suggest", "test", "fixtures") }
+  let(:config) { File.join(fixture_path, "sugg-conf", "suggest-bump.yaml") }
+
+  describe "when packaging the basic test fixture set" do
+    it "bump:patch version default" do
+      result = run_inspec_process("suggtool bump --suggest-config #{config}")
+      _(result.stderr).must_equal ""
+      _(result.exit_status).must_equal 0
+      _(result.stdout).must_include "Bumped:Patch version of"
+    end
```

Comment:
Ideally, we need to figure out a way to do this in a temporary directory, having called `package` on the set, which will create the metadata yml file. We don't want to modify the metadata yaml file in the fixture directory, since multiple tests will be accessing it in parallel.

Reply:
Done!